includes/orders/classes/class-refund-validator.php

```diff
@@ -354,10 +355,10 @@ private function validate_item_and_add_totals( $original_item, $amounts_to_refun
 
 			// But grab from specified amounts if available. It should always be available.
 			if ( isset( $amounts_to_refund[ $column_name ] ) ) {
-				$attempted_amount = $amounts_to_refund[ $column_name ];
+				$attempted_amount = floatval( $amounts_to_refund[ $column_name ] );
```

Comment:
Can you revert this? Just because `edd_sanitize_amount()` will already do it and I don't want to confuse ourselves over which one we actually need.